### PR TITLE
fix: reading anchor tx ID from blockinfo

### PR DESCRIPTION
**Pull request by jorgemmsilva**

### CODE REVIEW

File: tools/cluster/tests/rotation_test.go [hunk 1]
```diff
@@ -216,7 +216,7 @@ func TestRotationFromSingle(t *testing.T) {
 	case incCounterResult := <-incCounterResultChan:
 		require.NoError(t, incCounterResult)
 	case <-time.After(10 * time.Second):
-		t.FailNow()
+		t.Fatal()
```
Comment (dessaya) [inline tools/cluster/tests/rotation_test.go:219]: Just curious, what's the difference?

**Review by dessaya — APPROVED** [review]

**Review by muXxer — APPROVED** [review]

**Comment by muXxer** [comment]
Sorry that was my fault :sweat_smile:

File: tools/cluster/tests/rotation_test.go (hunk 1 above)
Comment (jorgemmsilva) [reply tools/cluster/tests/rotation_test.go:219]: I was trying to find where it was failing, from my experience fatal prints the call stack

**Comment by jorgemmsilva** [comment]
the only people that don't produce bugs are the ones that don't write code :smile:

**Pull request merged**